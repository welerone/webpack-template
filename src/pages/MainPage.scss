.example-title-text {
  font-size: 18px;
  color: lightgrey;
}
