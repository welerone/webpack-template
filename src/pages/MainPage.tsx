import * as styles from './MainPage.scss';

export function MainPage() {
  return <div className={styles.exampleTitleText}>Webpack template</div>;
}
