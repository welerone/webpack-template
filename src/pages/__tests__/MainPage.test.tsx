import { render } from '@testing-library/react';
import { MainPage } from '../MainPage';

describe('MainPage', () => {
  it('renders correctly', () => {
    const { asFragment } = render(<MainPage />);

    expect(asFragment()).toMatchSnapshot();
  });
});
