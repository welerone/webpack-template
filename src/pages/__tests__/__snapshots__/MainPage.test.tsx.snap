// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainPage renders correctly 1`] = `
<DocumentFragment>
  <div>
    Webpack template
  </div>
</DocumentFragment>
`;
